docs/content/latest/releases/whats-new/latest-release.md

```diff
@@ -24,6 +24,304 @@ Refer to [issue 8531](https://github.com/yugabyte/yugabyte-db/issues/8531) for d
+* [[8084](https://github.com/yugabyte/yugabyte-db/issues/8084)] Remove dependence of client to server TLS on node to node TLS.
+* [[8170](https://github.com/yugabyte/yugabyte-db/issues/8170)] Correct fix for removing tablet in ProcessTabletAssignment
+* [[8204](https://github.com/yugabyte/yugabyte-db/issues/8204)] [YBase] GetLoadMoveCompletionPercent returns an incorrect 100% if tservers haven't
+* [[8225](https://github.com/yugabyte/yugabyte-db/issues/8225)] Disable automatic tablet splitting for YEDIS tables
+* [[8234](https://github.com/yugabyte/yugabyte-db/issues/8234)] [DocDB] Remove tablet_split_heartbeat_data_provider and delegate all split decisions to yb-master
+* [[8249](https://github.com/yugabyte/yugabyte-db/issues/8249)] [YSQL] Allow any hash column in index to be an expression based column
+* [[8254](https://github.com/yugabyte/yugabyte-db/issues/8254)] No leader lease needed for BackfillIndex
+* [[8257](https://github.com/yugabyte/yugabyte-db/issues/8257)] [YBase] Disable tablet splitting for PITR tracked tables
+* [[8271](https://github.com/yugabyte/yugabyte-db/issues/8271)] YB-Master should not periodically read the pg_class table if there are no tablespaces.
+* [[8282](https://github.com/yugabyte/yugabyte-db/issues/8282)] Fix commons-lang deps.
+* [[8294](https://github.com/yugabyte/yugabyte-db/issues/8294)] Fix missing conflict when creating row using multiple inserts
+* [[8295](https://github.com/yugabyte/yugabyte-db/issues/8295)] [DocDB] Improved tablet RocksDBs shutdown process
+* [[8323](https://github.com/yugabyte/yugabyte-db/issues/8323)] [YSQL] Add support for USING INDEX TABLESPACE
+* [[8330](https://github.com/yugabyte/yugabyte-db/issues/8330)] [YCQL] Provide capability to skip writing null jsonb attribute in UPDATE stmt
+* [[8347](https://github.com/yugabyte/yugabyte-db/issues/8347)] Wait until executor finish executing async calls on shutdown
+* [[8382](https://github.com/yugabyte/yugabyte-db/issues/8382)] [YSQL] Import Fix mishandling of resjunk columns in ON CONFLICT ... UPDATE tlists.
+* [[8390](https://github.com/yugabyte/yugabyte-db/issues/8390)] Fix NPE Handling for indexed_table_id
+* [[8392](https://github.com/yugabyte/yugabyte-db/issues/8392)] [YSQL] Use negative cache elements to avoid read catalog entries from master
+* [[8473](https://github.com/yugabyte/yugabyte-db/issues/8473)] [YBase] Log the requested memory size in the assertion for allocation failure
+* [[8485](https://github.com/yugabyte/yugabyte-db/issues/8485)] Convert OperationCompletionCallback into std::function
+* [[8485](https://github.com/yugabyte/yugabyte-db/issues/8485)] Simplify MVCCManager to allow aborting only last operation
+* [[8485](https://github.com/yugabyte/yugabyte-db/issues/8485)] Unify ConsensusRound callbacks
+* [[8496](https://github.com/yugabyte/yugabyte-db/issues/8496)] Downgrade gperftools to 2.7
```

Comment:
bugfix -- I think we called this out in stable versions, but for latest, prolly not worth it